templates/advantage/index.html

```diff
@@ -445,6 +465,15 @@ <h2>Plans for enterprise use</h2>
       tab.parentNode.classList.remove('p-table--open');
       tab.querySelector('.p-icon--contextual-menu').classList.remove('u-rotate');
       panel.setAttribute('aria-hidden', 'true');
+      if(panel.id.includes("row")){
+          dataLayer.push({
+          'event' : 'GAEvent',
+          'eventCategory' : 'Advantage',
+          'eventAction' : 'table-row-collapse',
+          'eventLabel' : panel.id.split('-')[2],
```

Comment:
Can you set this to a variable that explains what it is, please? It's unclear currently.